setup.cfg

```diff
@@ -79,6 +79,7 @@ install_requires =
     typing_extensions>=4.2.0
     vispy>=0.13.0,<0.14
     wrapt>=1.11.1
+    zarr>=2.12.0 # needed by `builtins` (dask.array.from_zarr) to open zarr
```

Comment:
is there a way to depends on `dask[array]`, so that say if they update their dependencies it reflects here ?

Reply:
I don't think so?
`dask[array]` doesn't depend on zarr and `pip install "dask[array]"` in a fresh python 3.10:
```
Installing collected packages: zipp, toolz, pyyaml, packaging, numpy, locket, fsspec, cloudpickle, click, partd, importlib-metadata, dask
```

Reply:
Oh, ok I misunderstood.I see that `from_zarr` does a lazy import of `zarr` indeed.

I think as a desktop app for end users then it's reasonable to install zarr by default.

Reply:
Yes. for bundled version I will add zarr dependency